diff --git a/dist/browser/runtime/webcrypto.js b/dist/browser/runtime/webcrypto.js
index f9e1e915390c4ebf7cf0581f4b183cae253ae402..b0b94ae6ae35f36fb187b1d316d5a4eab880531d 100644
--- a/dist/browser/runtime/webcrypto.js
+++ b/dist/browser/runtime/webcrypto.js
@@ -1,2 +1,13 @@
 export default crypto;
-export const isCryptoKey = (key) => key instanceof CryptoKey;
+export const isCryptoKey = (key) => {
+  if (
+    "type" in key &&
+    "extractable" in key &&
+    "algorithm" in key &&
+    "usages" in key
+  ) {
+    return true;
+  }
+
+  return key instanceof CryptoKey;
+};