diff --git a/dist/browser/runtime/is_key_like.js b/dist/browser/runtime/is_key_like.js
index 8d35fdc84c4f2642dd144004a8faffca7efba578..cc8a9cd40940b65ab8c75bb854ab6432d70cf922 100644
--- a/dist/browser/runtime/is_key_like.js
+++ b/dist/browser/runtime/is_key_like.js
@@ -1,5 +1,18 @@
-import { isCryptoKey } from './webcrypto.js';
+import { isCryptoKey } from "./webcrypto.js";
 export default (key) => {
-    return isCryptoKey(key);
+  if (isCryptoKey(key)) {
+    return true;
+  }
+
+  if (
+    "type" in key &&
+    "extractable" in key &&
+    "algorithm" in key &&
+    "usages" in key
+  ) {
+    return true;
+  }
+
+  return false;
 };
-export const types = ['CryptoKey'];
+export const types = ["CryptoKey"];